### DISCUSSION TITLE
Question: Is it possible to apply workflow only to sub tree?

### DISCUSSION THREAD

**Discussion by bielu** [umbraco/Umbraco.Workflow.Issues#83] [Ideas]
Hi, 

I have question, when playing with workflow i noticed even though i apply workflow from one of descandants of my root, workflow was applied to whole tree, so based on this my question is "Is it possible to apply workflow only to subtree of content?".

**Comment by nathanwoulfe** [comment]
Hey @bielu that's absolutely how it should be working - configuration will inherit to all descendants, unless explicitly excluded (this feature does require a license)

For example, content tree could look like this:

- Root => no workflow
  - Child => no workflow
    - Grandchild => has workflow
      - Great grandchild  => inherits workflow
      - Great grandchild => inherits workflow (but can be excluded using excluded nodes setting)
    - Grandchild => no workflow

If you're seeing different behaviour, please share your configuration and I'll take a look.
